charts/operator/templates/_helpers.yaml

```diff
@@ -0,0 +1,121 @@
+{{ define "webhooks.clientConfig" -}}
+{{ printf "\n" }}
+{{- if .Values.webhooks.useCustomTlsCerts -}}
+caBundle: "{{ .Values.webhooks.caBundle | b64enc }}"
+{{ end -}}
+service:
+  name: airflow-operator-webhook-service
+  namespace: '{{ .Release.Namespace }}'
+  path: '{{ .path }}'
+{{- end }}
+
+# Declaring volumes for the Operator manager deployment, either use Certmanager-provided
+{{ define "manager.volumes" -}}
+{{ printf "\n" -}}
+{{- if and .Values.airgapped .Values.runtimeVersions.configMapName -}}
+- name: versions
+  configMap:
+    name: {{ .Values.runtimeVersions.configMapName }}
+{{- end }}
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) -}}
+- name: cert
+  secret:
+    defaultMode: 420
+    secretName: webhook-server-cert
+{{- else -}}
+- name: custom-certs
+  secret:
+    defaultMode: 420
+    secretName: webhooks-tls-certs
+{{- end -}}
+{{- end }}
+
+{{ define "manager.volumeMounts" -}}
+{{ printf "\n" -}}
+{{- if and .Values.airgapped .Values.runtimeVersions.configMapName -}}
+- mountPath: /etc/config/astro-runtime/metadata/versions.json
+  name: versions
+  subPath: versions.json
+{{- end }}
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) -}}
+- mountPath: /tmp/k8s-webhook-server/serving-certs
+  name: cert
+  readOnly: true
+{{- else -}}
+- mountPath: /tmp/custom-certs
+  name: custom-certs
+  readOnly: true
+{{- end -}}
+{{- end }}
+
+{{ define "manager.args" -}}
+{{ printf "\n" -}}
+- --health-probe-bind-address=:8081
+- --leader-elect
+- --metrics-bind-address={{ if .Values.manager.metrics.useSecuredEndpoint }}127.0.0.1{{ else }}0.0.0.0{{ end }}:8080
+{{- if gt (len .Values.airflowNamespaces) 0 }}
+- --namespaces={{ join "," .Values.airflowNamespaces }}
+{{- end -}}
+{{- if .Values.airgapped }}
+- --airgapped
+{{- end -}}
+{{- if .Values.openshift }}
+- --openshift
+{{- end -}}
```

Comment:
this should be added along with global.openshiftEnabled

Reply:
done